client/app/scripts/superdesk-monitoring/aggregate.js

```diff
@@ -335,19 +335,6 @@
             });
         };
 
-        /**
```

Comment:
No, I checked, this function is never called, take a look. there is function:
```     this.search = function(query) {
            this.searchQuery = query;
            _.each(this.groups, function(item) {
                item.query = query;
            });
            _.each(this.spikeGroups, function(item) {
                item.query = query;
            });
        };```
On reset it is called search(null), so no need for this.